.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "max-len": 0,
+    "no-unused-vars": 0,
```

Comment:
what is the reason to disable the rule?

Reply:
ESLint detects an error while function's parameters not yet exist in implementation.